test/game_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/card'
+require './lib/deck'
+require './lib/player'
+require './lib/turn'
+require './lib/game'
+
+class GameTest < Minitest::Test
+  def setup
+    @player1 = Player.new('Megan', @deck1)
+    @player2 = Player.new('Aurora', @deck2)
+    @game = Game.new(@player1, @player2)
+  end
+
+  def test_it_exists
+    assert_instance_of Game, @game
+  end
+
+  def test_it_has_attributes
+    assert_equal @player1, @game.player1
+    assert_equal @player2, @game.player2
+  end
+end
```

Comment:
Ideally, this test file would test every method in the Game class. I wasn't sure how to assert the other methods since I'm not yet comfortable with mocks/stubs, and the game relies on randomness.